security/s2a/internal/crypter/expander.go

```diff
@@ -0,0 +1,12 @@
+package crypter
+
+import (
+	"hash"
+)
+
+// hkdfExpander is the interface for the HKDF expansion function; see
+// https://tools.ietf.org/html/rfc5869 for details. It's use in TLS 1.3 is specified in
+// https://tools.ietf.org/html/rfc8446#section-7.2
+type hkdfExpander interface {
```

Comment:
If we're unexporting this, then we should also unexport `S2AAeadCrypter`. We can do that in a separate PR after #8 is submitted. WDYT?

Reply:
Sounds good. When I added `S2AAeadCrypter` I didn't realize that it would only be used in the `crypter` package. I'll unexport it in a separate PR.